frontend/lib/ui/dashboard/artefact_side_filters.dart

```diff
@@ -0,0 +1,37 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+import '../../providers/artefact_filters.dart';
+import '../../routing.dart';
+import '../side_filters.dart';
+import 'artefact_search_bar.dart';
+
+class ArtefactSideFilters extends ConsumerWidget {
+  const ArtefactSideFilters({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    final family = AppRoutes.familyFromContext(context);
+    final filters = ref.watch(artefactFiltersProvider(family));
+
+    return SizedBox(
+      width: SideFilters.width,
+      child: Column(
+        children: [
+          ArtefactSearchBar(),
+          const SizedBox(height: SideFilters.spacingBetweenFilters),
+          SideFilters(
+            filters: filters,
+            onOptionChanged: (filterName, optionName, value) => ref
```

Comment:
Small detail here, but should we pass the .handleFilterOptionChange function without any parameters like we do for the TestExecutionSideFilter to keep it more consistent and simpler?

Reply:
Another good catch